src/components/molecules/backItem.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<li class="menu__back-item back--hidden" id="back">
+			<p class="back-item__text">back</p>
+			<div class="menu__left-arrow-outline">
+					<img src="icons/arrow_left_black.svg" class="menu__arrow" alt="Black arrow left">
+			</div>
+	</li>
+</template>
+
+<script>
+	export default {
+		name: "back"
```

Comment:
I think I wouldn't make a whole seperate Vue component for a list item that has a back button, since the difference between this component and the generic `li` is so small. I think I would make a generic list item and then check props to see if we need to create/show a back button alongside the name or not. Not saying that that's the only way to go, but just a suggestion.